src/pages/home/components/nav-bar.tsx

```diff
@@ -1,10 +1,33 @@
 import { User } from 'networking/types/user';
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
+import { useDebounce } from 'hooks/useDebounce';
 import { AppLink, RouteName } from 'routes';
+import { UserContext } from '../../../common/user-context';
 import styles from './nav-bar.module.scss';
 
 const NavBar = () => {
   const activeUser = JSON.parse(localStorage.getItem('activeUser') ?? '') as User;
+  const contextValue = useContext(UserContext);
+  const [searchTerm, setSearchTerm] = useState<string>('');
+  const { unfilteredUsers } = contextValue;
```

Comment:
you could save a line here:
```suggestion
  const { unfilteredUsers }  = useContext(UserContext);
  const [searchTerm, setSearchTerm] = useState<string>('');
```